middleware/haproxy/detectors-haproxy.tf

```diff
@@ -61,7 +61,7 @@ resource "signalfx_detector" "session_limit" {
   name = format("%s %s", local.detector_name_prefix, "Haproxy session")
 
   program_text = <<-EOF
-    A = data('haproxy_session_current', filter=${module.filter-tags.filter_custom})${var.session_limit_aggregation_function}${var.session_limit_transformation_function}
+    A = data('haproxy_session_current', filter=${var.session_limit_filter} and ${module.filter-tags.filter_custom})${var.session_limit_aggregation_function}${var.session_limit_transformation_function}
```

Comment:
why not hardcode directly if not desirable?

Reply:
At first I thought this could be useful but after reading more documentation I  removed it.